s5_subwords/run.sh

```diff
@@ -0,0 +1,293 @@
+                                              > $subword_dir/text_corpus
+
+    # We again subword tokenize to create a subword lexicon
+    #python3 local/sw_methods/bpe/apply_bpe.py -i data/$lang/train/tokens \
+    #                                          --codes $subword_dir/pair_codes \
+    #                                          | sed 's/ /\n/g' | sort -u > $subword_dir/subwords
+  
+    elif [[ $method == 'unigram' || $method == 'sp_bpe' ]]; then
+    
+    model=$subword_dir/unigram_${sw_count}
+
+    # Form a given text corpus we learn to create subwords we store the "model" as in $model
+    python3 local/sw_methods/sp/train_sp.py -i $text_corpus \
+                                            -o $model \
+                                            -v $sw_count \
+                                            -t $method \
+                                            || error "Failed training a ${method} model"
+
+    # Using the model we subword tokenize the kaldi format text files
+    for x in train test ; do
+      cp data/$lang/$x/text data/$lang/$x/text.old
+      python3 local/sw_methods/sp/apply_sp.py -m $model \
+                                              -t $method \
+                                              --kaldi_text "True" \
+                                              -i data/$lang/$x/text.old > data/$lang/$x/text \
+                                              || error "Failed applying the ${method} model"
+    done 
+    python3 local/sw_methods/sp/apply_sp.py -m $model \
+                                            -t $method \
+                                            -i $text_corpus \
+                                             > $subword_dir/text_corpus
+
+    # Note: What text file should we use here, if we use the text_corpus the subword lexicon 
+    # might increase.  
+    #cp $subword_dir/text_corpus $subword_dir/text_corpus_plus_text
+    #cut -d" " -f2- data/$lang/train/text >> $subword_dir/text_corpus_plus_text
+    #cut -d" " -f2- data/$lang/train/text | sed 's/ /\n/g' | LC_ALL=C sort -u | grep -v '^[[:space:]]*$' > $subword_dir/subwords 
+    #cut -d" " -f2- $subword_dir/text_corpus | sed 's/ /\n/g' | sort -u | grep -v '^[[:space:]]*$' > $subword_dir/subwords 
+
+  elif [[ $method == 'morfessor' ]]; then
+    echo "To be done"
+  fi 
+  
+  #cut -d" " -f2- data/$lang/train/text | sed 's/ /\n/g' | grep -v '^[[:space:]]*$' | sort -u > $subword_dir/subwords 
+
+  # The following scripts should be run independant of which subword method is choosen
+  #echo "$0: Preparing lexicon"
+  #python3 local/prepare_lexicon.py --i $subword_dir/subwords \
+  #                                 --o $subword_dir/subword_lexicon \
+  #                                 --is_subword True
+
+  echo "$0: Preparing lexicon, dict folder and lang folder" 
+
+  cut -d" " -f2- data/$lang/train/text >> $subword_dir/text_corpus
+  local/prepare_dict_subwordV2.sh $subword_dir/text_corpus \
```

Comment:
If you only use these V2 scripts for sentence piece stuff, could they not we named _sp instead of V2? It would be more intuitive. Or are you using them instead of the ones with out a V2? Then I feel those should be overwritten

Reply:
Yeah, I will overwrite them.